specification/maps/data-plane/Route/stable/2025-01-01/route.json

```
@@ -0,0 +1,3783 @@
+        },
+        "isSimplifiedPolygon": {
+          "description": "Use this to specify whether simplified polygons are required, reducing the number of vertices while preserving the overall shape. Currently, isSimplifiedPolygon only supports true. High-definition polygons are supported in version 2024-07-01-preview",
+          "type": "boolean",
+          "default": "true"
+        },
+        "optimizeRoute": {
+          "description": "Specifies the parameter to use to optimize the route. If not defined, the default is \"fastestWithoutTraffic\" which returns the route to minimize the travel time without using current traffic information.\n\nExample: \"optimizeRoute\":\"shortest\"",
+          "type": "string",
+          "default": "fastestWithoutTraffic",
+          "enum": [
+            "shortest",
+            "fastestWithoutTraffic",
+            "fastestWithTraffic"
+          ],
+          "x-ms-enum": {
+            "name": "RouteRangeOptimizeRouteEnum",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "shortest",
+                "name": "Shortest",
+                "description": "The route is calculated to minimize the distance. Traffic information is not used."
+              },
+              {
+                "value": "fastestWithoutTraffic",
+                "name": "FastestWithoutTraffic",
+                "description": "Finds the fastest route, without factoring in traffic information."
+              },
+              {
+                "value": "fastestWithTraffic",
+                "name": "FastestWithTraffic",
+                "description": "The route is calculated to minimize the time using current traffic information. `Note`: Only supported for driving and truck travelMode."
+              }
+            ]
+          }
+        },
+        "avoid": {
+          "$ref": "#/definitions/RouteRangeAvoidEnum"
+        },
+        "vehicleSpec": {
+          "$ref": "#/definitions/RouteRangeVehicleSpec"
+        },
+        "distanceBudgetInMeters ": {
+          "description": "The distance budget specifies the maximum range in meters which can be traveled from the origin waypoint. It cannot be set when `timeBudgetInSec` is specified.\n\nWhen `isSimplifiedPolygon` is false, the maximum distance supported is 360,000 meters; otherwise, it is 500,000 meters.\n\nExample: \"distanceBudgetInMeters\":5000",
+          "type": "number",
+          "minimum": 0,
+          "maximum": 500000
+        },
+        "timeBudgetInSec": {
+          "description": "The time budget specifies the maximum time in seconds available for travel, defining how far one can go within this time constraint from the origin waypoint. It cannot be set when `distanceBudgetInMeters` is specified.\n\nWhen `isSimplifiedPolygon` is false, the maximum time supported is 14,400 seconds; otherwise, it is 21,600 seconds.\n\nExample: \"timeBudgetInSec\":3600",
+          "type": "number",
```

Schema with type: number should specify format 
```suggestion
          "type": "number",
          "format": "double",
```